### PR TITLE
New version: Microsoft.PowerShell version 7.3.2.0

### PR DESCRIPTION
**Pull request by anamnavi**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/95159)

### CODE REVIEW

File: manifests/m/Microsoft/PowerShell/7.3.2.0/Microsoft.PowerShell.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  InstallerType: wix
+  InstallerUrl: https://github.com/PowerShell/PowerShell/releases/download/v7.3.2/PowerShell-7.3.2-win-x86.msi
+  InstallerSha256: 43EBD047A0AC8951D01C59E8F42F27A50DA37556D779E99B4AFF009EF8418B5E
+  ProductCode: '{EE9CC6F4-ACF6-4B10-81BD-49D1C188770D}'
+- Architecture: arm
+  MinimumOSVersion: 10.0.17763.0
+  Platform:
+  - Windows.Universal
+  InstallerType: msix
+  InstallerUrl: https://github.com/PowerShell/PowerShell/releases/download/v7.3.2/PowerShell-7.3.2-win.msixbundle
```
Comment (Trenly) [inline manifests/m/Microsoft/PowerShell/7.3.2.0/Microsoft.PowerShell.installer.yaml:30]: The MSIXbundle packages are only intended for the MSStore source; Please update the PR to only include the msi packages

**Review by Trenly — CHANGES_REQUESTED** [review]

File: manifests/m/Microsoft/PowerShell/7.3.2.0/Microsoft.PowerShell.installer.yaml (hunk 1 above)
Comment (anamnavi) [reply manifests/m/Microsoft/PowerShell/7.3.2.0/Microsoft.PowerShell.installer.yaml:30]: Our team (PowerShell) is working to support msi for arm, but until then the temporary solution is to include the msix installer only for arm so this is needed in the installer manifest for now.

Comment (anamnavi) [reply manifests/m/Microsoft/PowerShell/7.3.2.0/Microsoft.PowerShell.installer.yaml:30]: @trenly Our team (PowerShell) is working to support msi for arm, but until then the temporary solution is to include the msix installer only for arm so this is needed in the installer manifest for now.

Comment (Trenly) [reply manifests/m/Microsoft/PowerShell/7.3.2.0/Microsoft.PowerShell.installer.yaml:30]: Ah, I see. I was only aware that it had caused issues in the past

**Review by mdanish-kh — CHANGES_REQUESTED** [review]
Do we need `Scope: machine` in the manifest? A user is having difficulty installing PowerShell in a per-machine context where winget pulls an older 7.2.7 version as the latest manifests do not include the Scope field and winget defaults to User scope.
The issue is being tracked here https://github.com/microsoft/winget-pkgs/issues/95172

**Review by ItzLevvie — APPROVED** [review]

**Comment by floh96** [comment]
@denelon sorry to ping you can you please look why the pipeline keeps failing for the powershell package?

**Comment by denelon** [comment]
@floh96 we're having intermittent issues with the pipelines. We've opened tickets and have been triggering runs to try and get things to successfully run. I'll trigger this one again.

**Review by stephengillie — APPROVED** [review]

**Pull request merged**